.chloggen/gbbr_opaque_exporters-azuredataexplorer.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Can I ask how is this breaking?

Reply:
I somewhat agree with you. Can you take this discussion to the issue please? That's where it was described as a breaking change, so we should have the discussion scoped to all of the changes in this category.

Some PRs were already merged like this btw.